garden_ai/gardens.py

```diff
@@ -93,30 +104,38 @@ class Garden(BaseModel):
         build one for the user with the datacite api.
     """
 
+    model_config = ConfigDict(validate_default=False)
```

Comment:
What's the difference between 
`class MyModel(BaseModel, **kwargs)` and `model_config = ConfigDict(**kwargs)`? if they're doing the same thing lets stick to one or the other. either one is fine but aesthetically I do think the inheritance kwargs are Cool

Reply:
The docs show example usage of `validate_assignment` as an inheritance kwarg, but overall, it looks like the general recommend practice is using the ConfigDict. So I think we just stick to using just that.